locales/zh_CN/LC_MESSAGES/scalar_tensor.po

```diff
@@ -191,14 +192,14 @@ msgstr ""
 
 #: ../../scalar_tensor.rst:99
 msgid ""
-"In Python, x[(exp1, exp2, ..., expN)] is equivalent to x[exp1, exp2, ...,"
-" expN]; the latter is just syntactic sugar for the former."
+"In Python, x[(exp1, exp2, ..., expN)] is equivalent to x[exp1, exp2, ..., "
+"expN]; the latter is just syntactic sugar for the former."
 msgstr ""
```

Comment:
Not trannslated?

Reply:
Oh,I just find a small mistake and want to change it,so I don't know why translator didn't translate these words before.(´･㉨･`)

Reply:
Oh I see, added #82 for this, thank for the contribution, merging now :)